apis/binding/v1alpha1/servicebinding_types.go

```diff
@@ -30,18 +30,6 @@ var templates = map[string]string{
 
 // ServiceBindingSpec defines the desired state of ServiceBinding.
 type ServiceBindingSpec struct {
-	// MountPath specifies the path inside the app container where bindings
-	// will be mounted.  The environment variable `SERVICE_BINDING_ROOT`
-	// has higher precedence than this option, and setting it within an
-	// application to be bound will cause MountPath to be ignored and will
-	// always mount resources as files.  If it isn't set, then it is
-	// automatically set to the value of MountPath.  If neither MountPath
-	// nor `SERVICE_BINDING_ROOT` are set, this field defaults to
-	// `/bindings`, and `SERVICE_BINDING_ROOT` is also set.  This results
-	// in the file being mounted under the directory specified.
-	// +optional
-	MountPath string `json:"mountPath,omitempty"`
```

Comment:
since the field is remove, `DeepCopy` function and CRD need to be regenerated. This is performed with `make generate manifests`

Reply:
Update: it already produced a new CRD.  Check the diff for `config/crd/bases/binding.operators.coreos.com_servicebindings.yaml`.